integrations/gcp/.port/spec.yaml

```diff
@@ -0,0 +1,27 @@
+type: gcp
+description: gcp integration for Port Ocean
+icon: GoogleCloud
+features:
+  - type: exporter
+    section: Cloud Providers
+configurations:
+  - name: parent
+    required: true
+    type: string
+    description: The Organization/Folder/Project you want the integration to fetch resources from.
+  - name: serviceAccountFileLocation
+    required: true
+    type: string
+    description: The location of the service account's key file.
```

Comment:
isnt there a default you can look for?

Reply:
Can you give a suggestion for what you mean by default?

Reply:
when azure integration runs it search for these parameters in places they apear on the azure machine

Please note that i want the integration to work with as less parameters as possible 
so anything u can discover by getting the gcp app env vars or files then do it yourself without requesting parameters